pulp_ansible/tests/functional/api/collection/v2/test_sync.py

```diff
@@ -126,7 +126,7 @@ def test_sync_with_version_equality(self):
             "---\n"
             "collections:\n"
             "- name: robertdebock.ansible_development_environment\n"
-            '  version: "==1.0.1"\n'
+            '  version: "1.0.1"\n'
```

Comment:
Is `==1.0.1` valid at all?

Reply:
Not entirely sure, but I believe it is valid

Reply:
If it's still valid, I'd propose we keep a test with it.